source/documentation/getting-started/ecr-setup.html.md.erb

```diff
@@ -109,7 +109,7 @@ In the `resources/ecr.tf` file, you should see this section:
   # github_repositories = ["my-repo"]
 ```
 
-To use this feature, uncomment the last line, and set the list of repositories in which you want the GitHub Actions Secrets to be created.
+To use this feature, uncomment the last line, and set the list of MoJ GitHub Organisation repositories in which you want the GitHub Actions Secrets to be created.
```

Comment:
Is it worth expanding out `MoJ GitHub Organisation` to `ministryofjustice GitHub organisation`, as MOJ technically has 3?